waspc/test/Analyzer/TestUtil.hs

```diff
@@ -0,0 +1,16 @@
+module Analyzer.TestUtil where
+
+import qualified Wasp.Analyzer.Parser as P
+import qualified Wasp.Analyzer.TypeChecker as T
+
+pos :: Int -> Int -> P.SourcePosition
```

Comment:
I created these helper functions in tests to help me with quickly constructing needed stuff. If tests grow more / become harder to maintain, for this piece of the code, we should look into other ways to make it easier to write them (more abstractions/helper for writing tests), since it is important that they remain maintainable.